### PR TITLE
Cleanup some PHPDoc leftovers and unused variables

### PR DESCRIPTION
**Pull request by MorrisJobke**
Found while looking through my local stashes (and those are from some PHPStorm code checks)

### CODE REVIEW

File: lib/private/Setup/PostgreSQL.php [hunk 1]
```diff
@@ -79,14 +79,13 @@ public function setupDatabase($username) {
 			$this->createDatabase($connection);
 			$query = $connection->prepare("select count(*) FROM pg_class WHERE relname=? limit 1");
 			$query->execute([$this->tablePrefix . "users"]);
-			$tablesSetup = $query->fetchColumn() > 0;
+			$query->fetchColumn() > 0;
```
Comment (ChristophWurst) [inline lib/private/Setup/PostgreSQL.php]: can't you just remove the statement, or maybe the full query? this seems to useless now

**Review by ChristophWurst — APPROVED** [review]
:+1: otherwise

File: lib/private/Collaboration/Collaborators/RemoteGroupPlugin.php [hunk 2]
```diff
@@ -38,8 +38,6 @@ class RemoteGroupPlugin implements ISearchPlugin {
 
 	/** @var ICloudIdManager */
 	private $cloudIdManager;
-	/** @var IConfig */
-	private $config;
```
Comment (blizzz) [inline lib/private/Collaboration/Collaborators/RemoteGroupPlugin.php]: remove IConfig from the uses, too, seems unused as well

**Review by blizzz — APPROVED** [review]
besides the now unnecessary use mentioned before

**Comment by MorrisJobke** [comment]
@ChristophWurst @blizzz Both resolved :)

**Review by ChristophWurst — APPROVED** [review]
Awesome :muscle:

**Pull request merged**